scripts/fancybox.js

```diff
@@ -17,9 +17,8 @@ hexo.extend.tag.register('fancybox', function(args){
 
   var title = args.join(' ');
 
-  return '<a data-fancybox="gallery" href="' + original + '" data-caption="' + title + '">' +
-    '<img src="' + (thumbnail || original) + '" alt="' + title + '">'
-    '</a>' +
-    (title ? '<span class="caption">' + title + '</span>' : '');
+  return `<a data-fancybox="gallery" href="${original}" data-caption="${title}">
+    <img src="${thumbnail || original}" alt="${title}">
+    </a>
+    ${title ? `<span class="caption">${title}</span>` : ''}`;
```

Comment:
Is the ES6 template compatible with most the browsers?

Reply:
Don't worry, it's supported by Node.js 10